js/content/store.js

```diff
@@ -3190,9 +3250,115 @@ let WishlistPageClass = (function(){
                 DynamicStore.clear();
                 location.reload();
             });
-        }, true)
+        }, true);
     }
 
+    WishlistPageClass.prototype.showModalDialog = function() {
```

Comment:
This function name should specify that the modal dialog is the "Export Wishlist" modal dialog and not some other modal dialog.

Reply:
Done